rag/multimedia_rag/video_rag.py

```
@@ -0,0 +1,400 @@
+            
+        except Exception as e:
+            logger.error(f"Failed to analyze video with vision model: {e}")
+            return f"Video analysis failed: {str(e)}"
+    
+    async def _extract_and_analyze_frames(self, video_path: str) -> str:
+        """
+        Extract key frames from video and analyze them
+        """
+        try:
+            import cv2
+            import base64
```

Ruff (0.12.2)

`base64` imported but unused

Remove unused import: `base64`

(F401)